src/App.css

```diff
@@ -1,3 +1,8 @@
+html{
+  background: url('./assets/img/background.jpg');
+  background-size: cover;
+}
```

Comment:
That's cool but it would probably be better as a property in a styled component as the root